### DISCUSSION TITLE
[go1.10] 支持go1.10版本的编译使用

### DISCUSSION THREAD

**Discussion by deanz-io** [emmansun/gmsm#101] [General]
因为某些场景下需要支持windows xp系统，而go支持windows xp的最后一个版本是go1.10，所以是否可以考虑支持go1.10版本？

**Comment by emmansun** [comment]
无此计划

**Comment by deanz-io** [comment]
go版本的升级会导致对一些旧的操作系统不再提供支持，比如windows，android，ios等。这意味着部分软件如果要兼容在旧的操作系统上运行，就没办法对go的版本进行持续升级，所以我认为gmsm应该谨慎考虑对go最低版本支持的能力，而不是一味的跟着go的版本进行持续的升级。

**Reply by emmansun** [reply]
首先感谢您的建议。有些东西是不可以兼得的，如果要使用低版本的golang，可以固定使用支持该版本的gmsm版本，当然版本策略可以改善一下，类似于gmsm/v2。
